config/local-development/tilt/replace-image.yaml

```diff
@@ -1,4 +1,4 @@
 - op: replace
-  path: /spec/template/spec/containers/1/image
+  path: /spec/template/spec/containers/0/image
```

Comment:
please look at the pki pull request on they improved on the tilt solution, this should not be needed.

Reply:
I did take a look and didn't see how this was resolved...

Reply:
I cannot accept this in this state, do you want to meet to fix it together?